stablehlo/tests/transforms/stablehlo_legalize_qdq_to_quantized_op.mlir

```diff
@@ -0,0 +1,151 @@
+  %0 = stablehlo.uniform_quantize %arg0 : (tensor<4x5xf32>) -> tensor<4x5x!quant.uniform<ui8:f32, 34.0:16>>
+  %1 = stablehlo.uniform_dequantize %0 : (tensor<4x5x!quant.uniform<ui8:f32, 34.0:16>>) -> tensor<4x5xf32>
+  %2 = stablehlo.uniform_quantize %arg1 : (tensor<4x5xf32>) -> tensor<4x5x!quant.uniform<ui8:f32, 34.0:16>>
+  %3 = stablehlo.uniform_dequantize %2 : (tensor<4x5x!quant.uniform<ui8:f32, 34.0:16>>) -> tensor<4x5xf32>
+  %4 = "stablehlo.map"(%1, %3) ({
+    ^bb0(%arg2: tensor<f32>, %arg3: tensor<f32>):
+    %ret = stablehlo.constant dense<2.0> : tensor<f32>
+    "stablehlo.return"(%ret) : (tensor<f32>) -> ()
+  }) {dimensions = array<i64: 0, 1>} : (tensor<4x5xf32>, tensor<4x5xf32>) -> tensor<4x5xf32>
+  %5 = stablehlo.uniform_quantize %4 : (tensor<4x5xf32>) -> tensor<4x5x!quant.uniform<ui8:f32, 34.0:16>>
+  func.return %5 : tensor<4x5x!quant.uniform<ui8:f32, 34.0:16>>
+}
+
+// -----
+
+// CHECK-LABEL failure_varidic_op
+// CHECK:       %0 = stablehlo.uniform_quantize %arg0 : (tensor<8x2xf32>) -> tensor<8x2x!quant.uniform<u8:f32, 3.400000e+01:16>>
+// CHECK-NEXT:  %1 = stablehlo.uniform_dequantize %0 : (tensor<8x2x!quant.uniform<u8:f32, 3.400000e+01:16>>) -> tensor<8x2xf32>
+// CHECK-NEXT:  %2 = stablehlo.uniform_quantize %arg1 : (tensor<2x2xf32>) -> tensor<2x2x!quant.uniform<u8:f32, 3.400000e+01:16>>
+// CHECK-NEXT:  %3 = stablehlo.uniform_dequantize %2 : (tensor<2x2x!quant.uniform<u8:f32, 3.400000e+01:16>>) -> tensor<2x2xf32>
+// CHECK-NEXT:  %4:2 = "stablehlo.all_gather"(%1, %3) {{.*}} : (tensor<8x2xf32>, tensor<2x2xf32>) -> (tensor<8x8xf32>, tensor<2x4xf32>)
+// CHECK-NEXT:  %5 = stablehlo.uniform_quantize %4#0 : (tensor<8x8xf32>) -> tensor<8x8x!quant.uniform<u8:f32, 3.400000e+01:16>>
+// CHECK-NEXT:  return %5, %4#1 : tensor<8x8x!quant.uniform<u8:f32, 3.400000e+01:16>>, tensor<2x4xf32>
+func.func @failure_varidic_op(%arg0: tensor<8x2xf32>, %arg1: tensor<2x2xf32>) -> (tensor<8x8x!quant.uniform<ui8:f32, 34.0:16>>, tensor<2x4xf32>) {
+  %0 = stablehlo.uniform_quantize %arg0 : (tensor<8x2xf32>) -> tensor<8x2x!quant.uniform<ui8:f32, 34.0:16>>
+  %1 = stablehlo.uniform_dequantize %0 : (tensor<8x2x!quant.uniform<ui8:f32, 34.0:16>>) -> tensor<8x2xf32>
+  %2 = stablehlo.uniform_quantize %arg1 : (tensor<2x2xf32>) -> tensor<2x2x!quant.uniform<ui8:f32, 34.0:16>>
+  %3 = stablehlo.uniform_dequantize %2 : (tensor<2x2x!quant.uniform<ui8:f32, 34.0:16>>) -> tensor<2x2xf32>
+  %4:2 = "stablehlo.all_gather"(%1, %3) {
+    all_gather_dim = 1 : i64,
+    channel_handle = #stablehlo.channel_handle<handle = 1, type = 0>,
+    replica_groups = dense<[[0, 2, 4, 6], [1, 3, 5, 7]]> : tensor<2x4xi64>
+  } : (tensor<8x2xf32>, tensor<2x2xf32>) -> (tensor<8x8xf32>, tensor<2x4xf32>)
+  %5 = stablehlo.uniform_quantize %4#0 : (tensor<8x8xf32>) -> tensor<8x8x!quant.uniform<ui8:f32, 34.0:16>>
+  func.return %5, %4#1 : tensor<8x8x!quant.uniform<ui8:f32, 34.0:16>>, tensor<2x4xf32>
+}
+
+// -----
+
+// CHECK-LABEL @failure_all_operands_not_quantized
```

Comment:
CHECK-LABEL name inconsistent with the actual function name

Reply:
corrected label name, ty.